swagger_server/controllers/submitter/rabbitmq_submitter.py

```diff
@@ -0,0 +1,57 @@
+from ons_ras_common import ons_env
+
+from pika import BlockingConnection, BasicProperties, URLParameters
+from pika.exceptions import AMQPError
+
+
+class RabbitMQSubmitter:
+
+    def __init__(self):
+        self.connection = None
+
+    def send_message(self, message, queue, tx_id=None):
+        """
+        Sends a message to rabbit mq and returns a true or false depending on if it was successful
+        :param message: The message to send to the rabbit mq queue
+        :param queue: the name of the queue
+        :param tx_id: transaction id
+        :return: a boolean value indicating if it was successful
+        """
+
+        ons_env.logger.info('sending message to queue')
+
+        properties = BasicProperties(headers={},
+                                     delivery_mode=2)
+        if tx_id:
+            properties.headers['tx_id'] = tx_id
+
+        try:
+            self.connection = self._get_connection()
+            channel = self.connection.channel()
+            channel.queue_declare(queue=queue)
+            published = channel.basic_publish(exchange='',
+                                              routing_key=queue,
+                                              body=message,
+                                              properties=properties)
+            return published
+        except AMQPError:
+            ons_env.logger.error("Rabbitmq message not sent")
+            return False
```

Comment:
The recommended pattern in Python is for error paths to return via exception, rather than return a falsey value.